app/templates/eligible.html

```diff
@@ -0,0 +1,22 @@
+{% extends "base.html" %}
+
+{%- from 'govuk_frontend_jinja/components/input/macro.html' import govukInput -%}
```

Comment:
Do we need this line?

Reply:
The components and their options can be found here https://govuk-frontend-jinja.herokuapp.com